runtimes/jaxonnxruntime/stable/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:20.04
+
+# Disable interactive installation mode
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install Python dependencies
+RUN apt-get update && apt-get install -y \
+    python3 \
+    python3-dev \
+    python3-pip && \
+    apt-get clean autoclean && apt-get autoremove -y
+
+RUN pip3 install --upgrade pip setuptools wheel
+
+# Copy local directories
+COPY ./test /root/test
+COPY ./setup /root/setup
+
+# Install test report dependencies
+RUN pip3 install --no-cache-dir -r /root/setup/requirements_report.txt
+
+############## ONNX Backend dependencies ###########
+ENV ONNX_BACKEND="jaxonnxruntime.backend"
+
+# Install dependencies
+RUN pip3 install --no-cache-dir \
+    jaxonnxruntime
```

Comment:
Good point!
There are many issues when I do tests. I fix all those them:
1. DEV_INFO has typo.  I fix it.
2. onnx-tf docker file was broken.  The test result (https://onnx.ai/backend-scoreboard/) fail long time. I fix it.
3. jaxonnxruntime request newer python since it use typing library.  I upgrade it to python3.9. Also I need upgrade pytest in the docker file.

Thanks